libraries/apollo-compiler/src/test/graphql/com/example/target_name/schema.graphqls

```diff
@@ -1,3 +1,5 @@
+extend schema @link(url: "https://specs.apollo.dev/kotlin_labs/v0.2", import: ["@targetName"])
+
```

Comment:
What about bumping the default kotlin_labs to v0.2 if no `@link` is found?

Reply:
Done in 5273b1504a840d1beb3478a1967ac6c685eb30d6!